app/src/main/java/com/example/dine_and_donate/HomeFragments/MapFragment.java

```diff
@@ -111,6 +111,8 @@
     private User mCurrentUser;
     private ArrayList<Event> mNearbyEvents;
 
+    private ArrayList<Event> nearbyEvents = new ArrayList<>();
```

Comment:
ah, looks like you have some merging issues, this should be mNearbyEvents above now